README.md

```diff
@@ -172,7 +172,7 @@ docker-compose up
 
 ## Step 5: Change to a new branch
 
-Create a new branch for each issue you work on. Doing all your work on topic branches leaves your repository's main branch (named `master`) unmodified and greatly simplifies keeping your fork in sync with the main project.
+Create a new branch for each issue you work on. Doing all your work on topic branches leaves your repository's main branch (named `main`) unmodified and greatly simplifies keeping your fork in sync with the main project.
```

Comment:
Is there a reason to change the master branch to main?

master has always been the git default & the standard.

Reply:
It's no longer the default
https://www.theregister.com/2020/06/15/github_replaces_master_with_main/#:~:text=GitHub%20will%20drop%20the%20term,t%20invoke%20actual%20human%20slavery.